### PR TITLE
Feature/2113/certificate status should change

### PR DESCRIPTION
**Pull request by Tolik170**
## Description

display used and expired status in my-certificates page

#### Screenshots
image
image

### Checklist

- [x] 🔽 My branch is up-to-date with "development" branch
- [x] ✅All tests passed locally
- [x] ✨My changes working with up-to-date admin and back-end part locally, like charm
- [x] 🔗 Link pull request to issue

### CODE REVIEW

**Review by maarekua — APPROVED** [review]

**Review by dmytruk-y — APPROVED** [review]

**Review by greentoss — APPROVED** [review]

File: src/containers/my-certificates/cetrificate-item/certificate-item.js [hunk 1]
```diff
@@ -2,64 +2,78 @@ import React from 'react';
...
+  const date = new Date().toISOString();
+  const daysRemaining = Date.parse(item.dateEnd) - Date.parse(date);
+  const isLessThanMonth = daysRemaining < 2678400000 && daysRemaining > 0;
 
   const dateHandler = (date) => date.slice(0, 10).split('-').reverse().join('/');
 
   return (
-    <TableRow className={styles.root} data-cy='certificate-item'>
-      <TableCell data-cy='certificate-item-img'>
+    <TableRow className={isLessThanMonth ? styles.rootRed : styles.root}>
```
Comment (maarekua) [inline src/containers/my-certificates/cetrificate-item/certificate-item.js:26]: Create Table component

**Review by maarekua — CHANGES_REQUESTED** [review]

File: src/containers/my-certificates/cetrificate-item/certificate-item.js (hunk 1 above)
Comment (Tolik170) [reply src/containers/my-certificates/cetrificate-item/certificate-item.js:26]: Created

File: src/containers/my-certificates/certificate-table/certificate-table.js [hunk 2]
```diff
@@ -19,20 +20,22 @@ const CertificateTable = ({ items, openModal }) => {
   );
 
   const headerItems = useMemo(
```
Comment (maarekua) [inline src/containers/my-certificates/certificate-table/certificate-table.js:22]: Header and body items should be handled inside of table component.

**Review by maarekua — CHANGES_REQUESTED** [review]

File: src/containers/my-certificates/cetrificate-item/certificate-item.js (hunk 1 above)
Comment (Tolik170) [reply src/containers/my-certificates/cetrificate-item/certificate-item.js:26]: Body items render component with many props, so i need to pass all this props to table from certificateTable and OrderTable component and also check if it is table for certificate or for cart.

File: src/containers/my-certificates/certificate-table/certificate-table.js (hunk 2 above)
Comment (Tolik170) [reply src/containers/my-certificates/certificate-table/certificate-table.js:22]: Body items render component with many props, so i need to pass all this props to table from certificateTable and OrderTable component and also check if it is table for certificate or for cart.

**Review by maarekua — APPROVED** [review]

**Review by MickhaelSh — APPROVED** [review]

**Pull request merged**